cosmo_tester/test_suites/image_based_tests/cfy_manager_test.py

```diff
@@ -33,6 +40,21 @@
 ''' % MQ_PASSWORDS_PATH
 
 
+@pytest.fixture(scope='function')
+def manager_5_1_0(request, ssh_key, module_tmpdir, test_config, logger):
+    hosts = Hosts(ssh_key, module_tmpdir, test_config, logger, request,
+                  bootstrappable=True)
+    hosts.instances[0].image_name = test_config['upgrade']['manager_image']
+
+    hosts.create()
+    hosts.instances[0].wait_for_ssh()
```

Comment:
You don't need to wait_for_ssh before calling bootstrap- it waits already.

Reply:
Cool, I'll drop it. I was using [this fixture](https://github.com/cloudify-cosmo/cloudify-system-tests/blob/master/cosmo_tester/test_suites/bootstrap_based_tests/conftest.py#L16) for reference, so maybe it should be changed there as well?

Reply:
That fixture doesn't bootstrap, which is why it does wait_for_ssh

Reply:
Makes sense.